### ISSUE TITLE
Game executable not right

### ISSUE THREAD

**Issue by UG-Asger** [torhus/monster-browser#75]
When conneting to an Urban terror server you get 
![image](https://user-images.githubusercontent.com/68121399/210275860-0448b72e-0b6c-4726-b34e-e5d6f4ebe500.png)

**Comment by UG-Asger** [comment]
Even though i have this in settings
![image](https://user-images.githubusercontent.com/68121399/210275953-896f4013-4687-459e-9c38-ab8298d6e5b6.png)

**Comment by UG-Asger** [comment]
Location of your ... should be a browsing to your executable option. Like click, browse and select .exe file, if you know what i mean :)

**Comment by torhus** [comment]
Right, it would be great if the UrT installer would save the path in the registry. That way MB would set the correct value by itself. But it makes sense to ask for the path, I can look into doing that.

For now, you need to click on the Edit button next to the game list, and set exePath for UrtT to the correct value. Sorry for the inconvenience.

**Comment by The-Gig** [comment]
The program GUI only allows to set one game binary location, the one used for _quake3_ (as it's the "main" id tech 3 game. Other defaults such as master server address, are quake3-related too).
To correctly set up MB for the other games, you need to edit mods.ini (the fields are explained in MB website). And I think many people using UrT-specific binaries is the reason why MB searches for those binaries instead of following the default q3 ones...

Maybe thorus might create a GUI to edit mods.ini graphically, but I guess that could be an _huge_ work...

**Comment by UG-Asger** [comment]
Guess you didn't understand what i wrote ;) There is a bug in the software! Look at the screenshots. The program does not use the exe path you set :)

**Comment by UG-Asger** [comment]
The rest was a request for making it better ;)

**Comment by torhus** [comment]
@UG-Asger: It uses the exePath setting from the `[Urban Terror 4.3]`  section of the configuration file, not the path in Settings. Click on Edit, and you will see :)

**Comment by UG-Asger** [comment]
Okay, so no option to just browse to the .exe file? You will have to edit manually in the config file? Not very user friendly in my opinion :)

**Comment by torhus** [comment]
Well, that file is basically part of the user interface...

**Comment by The-Gig** [comment]
I guess you might "comment out" the exePath line from UrT section in mods.ini and it should fall back to use the main (meant for q3) path from general settings...
